src/psyclone/tests/psyir/nodes/codeblock_test.py

```diff
@@ -133,6 +134,36 @@ def test_codeblock_get_symbol_names(parser):
     assert "that_is_true" in sym_names
 
 
+def test_codeblock_ref_accesses(parser):
+    '''Test that the reference_accesses() method works as expected.
+    '''
+    vai = VariablesAccessInfo()
+    reader = FortranStringReader('''
+    subroutine mytest
+      myloop: DO i = 1, 10_i_def
+        a = b + sqrt(c) + 1.0_r_def
```

Comment:
As mentioned above, I would like to know what happens with "sqrt" (or any call), it is a signature?, is it readwrite or unkonwn?